storages/jsonl-storage/src/error.rs

```diff
@@ -0,0 +1,42 @@
+use {gluesql_core::result::Error, std::fmt};
+
+pub trait ResultExt<T, E: ToString> {
+    fn map_storage_err(self) -> Result<T, Error>;
+}
+
+impl<T, E: ToString> ResultExt<T, E> for std::result::Result<T, E> {
+    fn map_storage_err(self) -> Result<T, Error> {
+        self.map_err(|e| e.to_string()).map_err(Error::StorageMsg)
+    }
+}
+
+pub trait OptionExt<T, E: ToString> {
+    fn map_storage_err(self, error: E) -> Result<T, Error>;
+}
+
+impl<T, E: ToString> OptionExt<T, E> for std::option::Option<T> {
+    fn map_storage_err(self, error: E) -> Result<T, Error> {
+        self.ok_or_else(|| error.to_string())
+            .map_err(Error::StorageMsg)
+    }
+}
+
+pub enum JsonlStorageError {
+    FileNotFound,
+    TableDoesNotExist,
+    ColumnDoesNotExist(String),
+}
```

Comment:
How about using `thiserror` crate rather than iml fmt::Display manually?
ref. https://docs.rs/thiserror/latest/thiserror/

e.g.
https://github.com/gluesql/gluesql/blob/3c07222834d422dddfb43cd00d5de894e9cd629f/core/src/executor/evaluate/error.rs#L8-L18